### ISSUE TITLE
vgform textarea doesn't work with large amount of text

### ISSUE THREAD

**Issue by Nv7-GitHub** [vugu/vugu#178]
**Describe the bug**
I pasted 40KB of text into a `vgform:Textarea`. When I pressed a button (most likely page re-rendering), I got the error:
```
Uncaught RangeError: offset is out of bounds
    at Uint8Array.set (<anonymous>)
    at HTMLTextAreaElement.f (eval at syscall/js.valueCall (wasm_exec.js:373), <anonymous>:100:71)
```
This happens a lot when pasting text into a Textarea with vugu, not just here. 

**Software Versions**
Vugu version: In go.mod it has vugu `v0.3.3`
Go version: 1.15
Browser: Chrome

**To Reproduce**
You can see this at [Nv7Haven](https://nv7haven.tk) [(source code)](https://github.com/Nv7-GitHub/nv7haven.tk). You can find this in multiple places, both when pasting a large note using [QwikNotes](https://nv7haven.tk/?page=notes) or after generating a large wordsearch with [WSGen](https://nv7haven.tk/?page=wsgen), pressing Copy Text, and pasting it into the [WordSearch solver](https://nv7haven.tk/?page=wordsearch). 

If possible, please include a concise code snippet which reproduces the issue (and excludes other unnecessary functionality, for clarity).

**Expected behavior**
I expect the Textarea to work

**Comment by bradleypeabody** [comment]
Yeah it looks like there is a hard limit around 16k: https://github.com/vugu/vugu/blob/master/domrender/renderer-js-script.js#L809

That code can probably be rewritten to allocate a new larger buffer of the necessary size if and when it encounters larger event input.

I'll leave this open for now.  PRs welcome.

**Comment by owenwaller** [comment]
Closing this as the solution was merged as PR #190
